packages/amplify-console-hosting/hosting/index.js

```diff
@@ -0,0 +1,186 @@
+const constants = require('../constants/plugin-constants');
+const pathManager = require('../utils/path-manager');
+const fs = require('fs-extra');
+const utils = require('../utils/amplify-context-utils');
+const configUtils = require('../utils/config-utils');
+const questions = require('../modules/questions/question-generator');
+const ValidationError = require('../error/validation-error').default;
+const clientFactory = require('../utils/client-factory');
+const ora = require('ora');
+const tableUtis = require('../utils/table-utils');
+
+const HELP_INFO_PLACE_HOLDER =
+  'Manual deployment allows you to publish your web app to the Amplify Console without connecting a Git provider. Continuous deployment allows you to publish changes on every code commit by connecting your GitHub, Bitbucket, GitLab, or AWS CodeCommit repositories.';
+const REMOVE_ERROR_MESSAGE = 'There was an error removing the auth resource';
+const HOSTING_NOT_ENABLED = 'Amplify console hosting is not enabled!';
+const HOSTING_ALREADY_ENABLED =
+  'Amplify Console hosting has already been enabled';
+const META_FILE_VALIDATION =
+  'Amplify console hosting information is missed from amplify meta file';
+const FRONTEND_EXISTED_WARNING =
+  'Branches has already been added to amplify app. Can not enable local host';
```

Comment:
You have already connected branches to your Amplify Console app. Please visit the Amplify Console to manage your branches.